docs/core/diagnostics/diagnosticsource-getting-started.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Getting Started with DiagnosticSource
+description: A tutorial to create a basic DiagnosticSource and understand key concepts
+ms.date: 05/02/2022
+---
+# Getting Started with DiagnosticSource
+
+**This article applies to: ✔️** .NET Core 3.1 and later versions **✔️** .NET Framework 4.5 and later versions
+
+This walkthrough shows <xref:System.Diagnostics.DiagnosticSource?displayProperty=nameWithType> 
```

Comment:
Do we expect readers to understand what the purpose of DiagnosticSource is at this point? In the EventSource docs there was an 'overview' page before the 'getting started' page that had a paragraph describing what EventSource was and what it could be useful for. We don't have to rigidly follow the same structure, but somewhere very early in the documentation devs will probably want to understand this.